browser-interface/packages/shared/friends/ensureFriendProfile.ts

```diff
@@ -1,7 +1,10 @@
-import { Avatar } from '@dcl/schemas'
-import { ProfileAsPromise } from 'shared/profiles/ProfileAsPromise'
-import { ProfileType } from 'shared/profiles/types'
+import type { Avatar } from '@dcl/schemas'
+import { retrieveProfile } from 'shared/profiles/retrieveProfile'
 
 export function ensureFriendProfile(userId: string): Promise<Avatar> {
-  return ProfileAsPromise(userId, undefined, ProfileType.DEPLOYED) // Friends are always deployed ATM
+  /**
+   * Request the `DEPLOYED` version of the profile. This fetches the profile
+   * from the catalyst, not from comms.
+   */
```

Comment:
Requesting from comms was a purposeful optimizations to lower the load in the servers

Reply:
I'll move the comment to make it clear that this is still the case